src/main/java/edu/cornell/kfs/concur/dto/ConcurStandardAccountingExtractDTO.java

```diff
@@ -0,0 +1,6 @@
+package edu.cornell.kfs.concur.dto;
+
+public class ConcurStandardAccountingExtractDTO {
```

Comment:
I had a conversation with Nancy on this and we found a paradigm Bryan used with AchIncomeDelimitedFlatFileSpecification, which uses a business object package inside batch.  I think that works well, and will switch to that.